src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -98,6 +98,28 @@ private void PrintHelp()
       bicep build file.bicep --outfile file.json
       bicep build file.bicep --no-restore
 
+    {exeName} format [options] <file>
+    Formats a .bicep file.
+
+    Arguments:
+      <file>        The input file
+
+    Options:
+      --outdir <dir>        Saves the output at the specified directory.
+      --outfile <file>      Saves the output as the specified file path.
+      --stdout              Prints the output to stdout.
+      --newline             Set newline char. Valid values are ( Auto | LF | CRLF | CR ).
```

Comment:
>CR [](http://example.com/codeflow?start=84&length=2)

This is used by pre-OSX Macs. Can you double check we still support it in the parser? We supported it originally, but I'm not sure whether we took it out since.

Reply:
+1, I agree we should drop `CR` from this list.

VSCode doesn't give it as an option, for example:
<img width="638" alt="image" src="https://user-images.githubusercontent.com/38542602/193976251-20bc095d-f06c-481f-910f-ed5c011af911.png">

Reply:
Makes absolute sense!

Reply:
@shenglol - any concerns with removing `CR` from the `NewlineOption` enum?

Reply:
I think the other option we have here is just not giving the user the choice, and always using `Auto`. AFAIK that's what we do in the [Language Server implementation](https://github.com/Azure/bicep/blob/4863c9c5799af0b8679ca10e41b8df6662146c32/src/Bicep.LangServer/Handlers/BicepDocumentFormattingHandler.cs#L45), so there's an argument to be made for consistency.

Reply:
I'm all for defaulting to Auto if you are OK with that

Reply:
I think it would be better to keep `CR` to make the formatter compatible with EditorConfig. Also, `CR` is the line ending character for macOS 9 and earlier. Removing it will result in a breaking change for users who are still on old mac systems.

Reply:
Defaulting to Auto seems a good idea.

Reply:
So keep it as is then?

Reply:
Yep.

Reply:
Yeah